### ISSUE TITLE
Update example inputs in caffe_inputs

### ISSUE THREAD

**Issue by zhangce** [HazyResearch/CaffeConTroll#49]
sgeadmin@master:~/CaffeConTroll$ ./caffe-ct train caffe_inputs/lenet_solver.prototxt
Start writing 60000 to toprocess.bin...
Finished writing images to toprocess.bin...
Constructing CONV layer with Grouping = 1 (# Input Grouping=1)
Constructing MAXPOOLING (# Input Grouping=1)
Constructing CONV layer with Grouping = 1 (# Input Grouping=1)
Constructing MAXPOOLING (# Input Grouping=1)
Constructing FC layer (# Input Grouping=1)
Constructing RELU layer (# Input Grouping=1)
Constructing FC layer (# Input Grouping=1)
Constructing SOFTMAX layer (# Input Grouping=1)
EPOCH: 0
caffe-ct: src/sched/../util.h:97: static float Util::get_learning_rate(std::string, float, float, float, float, float, float): Assertion `false' failed.
